src/dso_api/dynamic_api/serializers.py

```diff
@@ -145,6 +145,16 @@ def build_property_field(self, field_name, model_class):
     def to_representation(self, validated_data):
         data = super().to_representation(validated_data)
 
+        # URL encoding of URLfield content, i.e. spaces to %20
+        schema_fields = dict(self.fields)        
+        for field_name, field_value in schema_fields.items():
```

Comment:
Schema fields hebben alle namen in snake_case, maar `data` hier is al camelCase.